### DISCUSSION TITLE
Return Discounted Price also on getListingsByShop API

### DISCUSSION THREAD

**Discussion by IamSantosh** [etsy/open-api#1147] [Q&A]
Hi there,

I am facing one issue regarding getListingsByShop API where one of our Users registered with our Etsy App want us to fetch discounted price also.

With this API, we can easily fetch the Actual Price, but I also want the discounted price (if any) as this API only returns the original price.

"price": {
    "amount": 0,
    "divisor": 0,
    "currency_code": "string"
}

Below, I've also attached the screenshot related to the same.
![Screenshot from 2023-11-06 15-14-23](https://github.com/etsy/open-api/assets/19526862/6841ec45-34c2-4519-a97a-017e74dadb61)

**Comment by wessells** [comment]
There is no way to get the discounted price from the API. You can only get the discounted price from receipts for which you need both an API key and the oauth2.
